components/_patterns/03-organisms/slideshow/slideshow.twig

```diff
@@ -45,7 +45,7 @@
     {# Previous Arrow #}
     <div {{ bem('nav', (['prev']), slideshow__base_class) }}>
       {% include "@atoms/images/icons/_icon.twig" with {
-        icon_name: 'arrow',
+        icon_name: 'arrow-xsmall-thin',
```

Comment:
Should the "next" arrow be updated to use this too?

Reply:
Yes—I've gone and updated this.